docs/blueprints/pydantic.py

```diff
@@ -1,36 +1,29 @@
 from drf_spectacular.extensions import OpenApiSerializerExtension
 from drf_spectacular.plumbing import ResolvedComponent
 
+from pydantic import model_schema
```

Comment:
this import is invalid for me. this worked for me for:

`from pydantic.schema import model_schema`

Reply:
Thanks for catching this. I just committed that change!